src/signals/incident/components/IncidentPreview/components/Map/index.js

```diff
@@ -1,50 +1,58 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Map from 'components/Map';
 
 import { formatAddress } from 'shared/services/map-location';
+import MAP_OPTIONS from 'shared/services/configuration/map-options';
 import './style.scss';
+import Map from 'components/Map';
+import styled from '@datapunt/asc-core';
+
+const StyledMap = styled(Map)`
+  width: 100%;
```

Comment:
Shouldn't `100%` be set by default? I've seen this value set already a couple of times.

Reply:
I moved the defaults to the Map component